src/Service.Host/.eslintrc.json

```diff
@@ -11,6 +11,7 @@
         "react/jsx-filename-extension": 0,
         "spaced-comment": "off",
         "react-hooks/rules-of-hooks": "error",
-        "react-hooks/exhaustive-deps": "error"
+        "react-hooks/exhaustive-deps": "error",
+        "react/destructuring-assignment": ["always", { "ignoreClassFields": true }]
```

Comment:
Not really sure what this is... Is there somewhere you thought this rule had to be violated? 
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md

Reply:
It should actually be removed, I was trying to get something working but it's no longer needed to run without warnings